test/testng/org/openjdk/jextract/test/toolprovider/unsupported/TestUnsupportedTypes.java

```diff
@@ -66,6 +66,7 @@ public static Object[][] cases() {
             {"accepts_undefined_func",           REASON_UNSUPPORTED_TYPE},
             {"GLOBAL_UNDECLARED",                REASON_UNSUPPORTED_TYPE},
             {"undefined_typedef",                REASON_UNSUPPORTED_TYPE},
+            {"LONG_DOUBLE_PI",                   REASON_UNSUPPORTED_TYPE}
```

Comment:
watch out - this will only be unsupported on windows? Maybe better to use __128 or such?

Reply:
Okay. Changed to use __int128 constant